providers/supermicro/supermicro.go

```diff
@@ -450,20 +443,27 @@ type serviceClient struct {
 	sum       *sum.Sum
 }
 
-func newBmcServiceClient(host, port, user, pass string, client *http.Client) (*serviceClient, error) {
+func newBmcServiceClient(host, port, user, pass string, client *http.Client) *serviceClient {
 	sc := &serviceClient{host: host, port: port, user: user, pass: pass, client: client}
 
 	if !strings.HasPrefix(host, "https://") && !strings.HasPrefix(host, "http://") {
 		sc.host = "https://" + host
 	}
 
+	// sum is only for firmware related operations. Failing the client entirely because of a sum error
```

Comment:
The sum code can be removed entirely since bmclib was 'hard' forked (for the environment where sum was required to run remotely). I'll get round to cleaning this up sometime soon.